docs/services.md

```diff
@@ -6,7 +6,7 @@ using the Docker Engine API. Note that the engine needs to be part of a
 
```

Comment:
There's a "user guide" part that we'd like to preserve I think - let me move the relevant bits to a different folder and we can figure out converting to RST / where to include at a later time.